pkgs/development/python-modules/starlette/default.nix

```diff
@@ -46,12 +45,14 @@ buildPythonPackage rec {
   checkInputs = [
     aiosqlite
     databases
+    pytest-asyncio
     pytestCheckHook
     typing-extensions
   ];
 
-  pytestFlagsArray = [ "--ignore=tests/test_graphql.py" ];
-
+  disabledTestFiles = [ "tests/test_graphql.py" ];
+  # https://github.com/encode/starlette/issues/1131
```

Comment:
This issue was closed 10 days ago. Is the test working now?

Reply:
I updated the package to 0.14.2 and dropped the disable.

Reply:
Nevermind, this breaks fastapi because they *still* haven't gotten rid of UJSONResponse.